_posts/0000-01-02-krenee411.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
You can use the left and right arrows to navigate through the slides

Reply:
This is a great idea. It lets the user know they can easily move back and forth between the slides.